experimental/WreathMacdonaldpols/src/wreath_macdonald_polynomials.jl

```diff
@@ -0,0 +1,357 @@
+# I would like to credit Dario Mathiä who produced an initial version of the following code
+
+import Chevie: complex_reflection_group, charinfo, CharTable, CyclotomicNumbers
+using Oscar
+import Nemo: IntegerUnion
+
+"""
```

Comment:
```suggestion
export wreath_macs

"""
```
see the comment in `export.jl` for context

Reply:
Done here [99b12f3](https://github.com/oscar-system/Oscar.jl/pull/4797/commits/99b12f3bf044c120e2234dd1e32334cddb390ba8).